casepro/statistics/migrations/0008_dailyminutetotalcount.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('statistics', '0007_populate_label_totals'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='DailyMinuteTotalCount',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('item_type', models.CharField(help_text='The thing being counted', max_length=1)),
+                ('scope', models.CharField(help_text='The scope in which it is being counted', max_length=32)),
+                ('count', models.IntegerField()),
+                ('minutes', models.IntegerField()),
```

Comment:
I think due to time restrains we'll keep it as is, and take the technical debt of having to add/change it to seconds if we need to measure something more accurately in the future.